cmd/utils/dev.go

```diff
@@ -223,15 +223,33 @@ func SelectDevFromManifest(manifest *model.Manifest, selector OktetoSelectorInte
 	return manifest.Dev[devKey], nil
 }
 
+// YesNoDefault specifies what will be assumed when the user doesn't answer explicitly
+type YesNoDefault string
+
+const (
+	YesNoDefault_Unspecified = "[y/n]"
+	YesNoDefault_Yes         = "[Y/n]"
+	YesNoDefault_No          = "[y/N]"
+)
+
 // AskYesNo prompts for yes/no confirmation
-func AskYesNo(q string) (bool, error) {
+func AskYesNo(q string, d YesNoDefault) (bool, error) {
 	var answer string
 	for {
-		oktetoLog.Question(q)
-		if _, err := fmt.Scanln(&answer); err != nil {
+		oktetoLog.Question(fmt.Sprintf("%s %s: ", q, d))
+		_, err := fmt.Scanln(&answer)
+		if err != nil && err.Error() != "unexpected newline" {
```

Comment:
`fmt.Scanl` will try to store a value on every argument it gets passed, and will error out if it can't.

In this case we are passing a single argument, and an empty response from the user will cause an error as not even a single value can be scanned. Unfortunately, no sentinels or error constants are used in the `fmt` package so the only way to identify that error is to compare its string representation.